.github/issue_template.md

```diff
@@ -0,0 +1,46 @@
+<!-- Thanks for your interest in Spring Cloud Data Flow! Before you create a new issue, please answer the following question.-->
+
+**Is this a "Bug Report" or "Feature Request"?** (choose one and remove the irrelevant sections before you submit the issue):
```

Comment:
That would be more friendly and potentially less confusing. It would also save use the trouble of people forgetting to remove one chunk or the other and making a cluttered issue. Better all around.

Reply:
I tried sticking to a model that was successful in Kuberneres ecosystem, but if the newer workflow gives better guidance, we can try it. I will close this PR.